exercises/concept/annalyns-infiltration/.meta/design.md

```diff
@@ -0,0 +1,21 @@
+# Design
+
+## Learning objectives
+
+- Know of the existence of the `bool` type and its two values.
+- Know about boolean operators and how to build logical expressions with them.
+- Know of the boolean operator precedence rules.
```

Comment:
As far as I can tell, the concept introduction does not explain the precedence rules. Let's add that.